### PR TITLE
Adds Typescript support

### PR DESCRIPTION
**Pull request by webketje**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
+export interface Options {
+	/**
+   * Favor accuracy over speed.
+   * @default false
+   */
```
Comment (sindresorhus) [inline index.d.ts:5]:
```suggestion
	/**
	Favor accuracy over speed.
	
	@default false
	*/
```

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	accurate?: boolean;
+
+	/**
+   * Set a quality preset.
+   * @default 'medium'
+   */
+	quality?: 'low'|'medium'|'high'|'veryhigh';
```
Comment (sindresorhus) [inline index.d.ts:12]:
```suggestion
	quality?: 'low' | 'medium' | 'high' | 'veryhigh';
```

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: index.d.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
+export type Options = {
+	/**
+  Favor accuracy over speed.
+
+  @default false
+  */
```
Comment (sindresorhus) [inline index.d.ts:6]: Incorrect indentation on all of these.

Comment (webketje) [reply index.d.ts]: I guessed you favor tabs by looking at https://github.com/imagemin/imagemin-pngquant/blob/main/index.d.ts. Think it should be fine now. Would be nice to have prettier (or alternative) to format automatically.

**Pull request merged**